rana_qgis_plugin/libs/bridgestyle/qgis/togeostyler.py

```diff
@@ -1036,8 +1036,9 @@ def saveSpritesSheet(icons, folder):
     img, img2x, painter, painter2x, spritesheet, spritesheet2x = initSpriteSheet(width, height)
     x = 0
     for name, _sprites in sprites.items():
+        name_without_ext = os.path.splitext(name)[0]
```

Comment:
This function `saveSpritesSheet` appears to be unused by us, so I don't think you should change it